lib/ansible/modules/cloud/azure/azure_rm_storageaccount.py

```diff
@@ -158,7 +158,10 @@ class AzureRMStorageAccount(AzureRMModuleBase):
     def __init__(self):
 
         self.module_arg_spec = dict(
-            account_type=dict(type='str', choices=[], aliases=['type']),
+            account_type=dict(
```

Comment:
As @mattclay [mentioned](https://github.com/ansible/ansible/pull/43812#issuecomment-411986915), the docker container used for validation does not have azure.mgmt.trafficmanager. This will cause sanity test failure on [azure_rm_storageaccount](https://github.com/ansible/ansible/pull/43812#issuecomment-412044096). We made this change to azure_rm_storageaccount.py to pass CI temporarily.

Reply:
Yep that's ok, you can always raise a 2nd PR to fix the choices sanity test and then rebase this based on the changes after that. But that's not strictly necessary.